### DISCUSSION TITLE
Add information about Bibliography

### DISCUSSION THREAD

**Discussion by bwbohl** [music-encoding/music-encoding.github.io#249] [Ideas]
It's great to have a bibliography listing related MEI items. Nevertheless as the Heading "**Selected** Bibliography" suggests there is no claim of completeness…

BUT:
The why and how of how items make it on that list is not clearly communicated. This should definitely be changed.
A follow up question for me is: Should we improve the ways of contributing to the bibliography e.g. via reference management software?

**Comment by musicEnfanthen** [comment]
Not sure if that is really an issue...? The "why and how" is quite straightforward. Anyone who wants to add an entry is welcome to do so by adding a bibtex entry to the [mei_bibliography.bib](https://github.com/music-encoding/music-encoding.github.io/blob/master/resources/mei_bibliography.bib) file, or at least to ask for support in doing so via one of the known communication channels. This may have been handled differently in the past, but from today's perspective, there shouldn't be too many restrictions on admission, apart from the fact that it should be somehow related to MEI. 

Do you think a short paragraph under the bibliography header would help here?

Indeed, it is called "Selected" because, as you point out, there is no claim to completeness, especially with older publications. (BTW, we still have to decide about additional items from the old Word list, see https://github.com/music-encoding/music-encoding.github.io/issues/21)

For your follow up question: What are you thinking about?

**Comment by ahankinson** [comment]
If it's in a .bib file, it can be managed with most reference managers.

**Comment by bwbohl** [comment]
Concerning reference managers I thought to reactivate the Zotero Group that was create a long tome ago and never really populated as it would also add a web interface for adding or editing items but that would mean to move the bibliography to Zotero and removing the bib file from this repo.
The Zotero Library can be consumed by bibbase and the integration into the website would practically stay the same.

Maybe for now sticking to the bibfile and adding a short paragraph a the top of the bibliography page is the most feasible.

**Comment by musicEnfanthen** [comment]
I see. So who would have access (and how) to the Zotero Group?

**Comment by bwbohl** [comment]
maybe we should move that to a discussion?
